doc/dev/kubernetes.rst

```diff
@@ -166,40 +171,33 @@ build, and the Ceph build too.
 5. Run a Rook cluster
 =====================
 
-.. note::
-
-    This is just some basic instructions: the Rook documentation
-    is much more expansive, at https://github.com/rook/rook/tree/master/Documentation
+Please refer to `Rook's documentation <https://github.com/rook/rook/tree/master/Documentation>`_
+for setting up a Rook operator, a Ceph cluster and the toolbox.
 
 The Rook source tree includes example .yaml files in
-``cluster/examples/kubernetes/ceph/``.  The important ones are:
-
-- ``operator.yaml`` -- runs the Rook operator, which will execute any other
-  rook objects we create.
-- ``cluster.yaml`` -- defines a Ceph cluster
-- ``toolbox.yaml`` -- runs the toolbox container, which contains the Ceph
-  CLI client.
-
-Copy these into a working directory, and edit as necessary to configure
+``cluster/examples/kubernetes/ceph/``. Copy these into
+a working directory, and edit as necessary to configure
 the setup you want:
 
-- Ensure that the ``image`` field in the operator matches the built Ceph image
+- Ensure that the ``image`` field in the ``operator.yaml`` matches the built Ceph image
```

Comment:
The image in the operator.yaml doesn't need to match since the ceph version is decoupled. You should just be able to use the `master` tag, which is already the default in the master branch. So it seems we can delete this line.